OpenOversight/tests/test_routes.py

```diff
@@ -606,3 +608,55 @@ def test_user_cannot_change_password_if_they_dont_match(mockdata, client,
         )
 
         assert 'Passwords must match' in rv.data
+
+
+def test_admin_can_add_police_department(mockdata, client, session):
+    with current_app.test_request_context():
+        login_admin(client)
+
+        form = DepartmentForm(name='Test Police Department',
+                              short_name='TPD')
+
+        rv = client.post(
+            url_for('main.add_department'),
+            data=form.data,
+            follow_redirects=True
+        )
+
+        assert 'New department' in rv.data
+
+
+def test_admin_cannot_add_duplicate_police_department(mockdata, client,
+                                                      session):
+    with current_app.test_request_context():
+        login_admin(client)
+
+        form = DepartmentForm(name='Chicago Police Department',
+                              short_name='CPD')
+
+        rv = client.post(
+            url_for('main.add_department'),
+            data=form.data,
+            follow_redirects=True
+        )
```

Comment:
I forget; this gets auto-cleaned between runs, right?

Reply:
You are correct - the `teardown()` in the `mockdata` pytest fixture in `conftest.py` will remove the data in the database between tests.